src/model/Computer.ts

```diff
@@ -0,0 +1,75 @@
+import { Random } from '@woowacourse/mission-utils';
+import { GAME_SIGN } from '../constants/game';
+import { VALIDATION } from '../constants/validation';
+
+interface Count {
+  strike: number;
+  ball: number;
+}
+
+class Computer {
+  #numbers;
+
+  constructor() {
+    this.#numbers = this.setNumbers();
+  }
+
+  setNumbers = () => {
+    const computer: number[] = [];
+
+    while (computer.length < VALIDATION.LENGTH) {
+      const number = Random.pickNumberInRange(
+        VALIDATION.MIN_NUMBER,
+        VALIDATION.MAX_NUMBER
+      );
+      if (!computer.includes(number)) {
+        computer.push(number);
+      }
+    }
+
+    return computer;
+  };
+
+  compareNumbers = (playerNumbers: number[]): Count => {
+    const count = { strike: 0, ball: 0 };
+
+    playerNumbers.forEach((number, index) => {
+      const isStrike = number === this.#numbers[index];
+      const isBall = this.#numbers.includes(number);
+
+      if (isStrike) {
+        count.strike += 1;
+      }
+
+      if (isBall && !isStrike) {
+        count.ball += 1;
+      }
+    });
```

Comment:
현재 로직을 보면 `isBall`이 `true` 일 때  ball 일수도 있고 strike일수도 있습니다.  즉 변수명이 부정확합니다. 
따라서 변수명을 교체하거나 코드를 아래처럼 변경하면 좋을 것 같습니다. 

```suggestion
    playerNumbers.forEach((number, index) => {
      const isStrike = number === this.#numbers[index];
      const isBall = this.#numbers.includes(number) && !isStrike

      if (isStrike) {
        count.strike += 1;
      }

      if (isBall) {
        count.ball += 1;
      }
    });
```